src/snowflake/snowpark/modin/pandas/series.py

```diff
@@ -2752,8 +2752,6 @@ def _update_inplace(self, new_query_compiler):
         # Propagate changes back to parent so that column in dataframe had the same contents
         if self._parent is not None:
             if self._parent_axis == 0:
```

Comment:
I removed the `else`, because it doesn't seem like `self._parent_axis` can be anything other than 1?

```python
In [2]: df = pd.DataFrame([[1, 2, 3], [4, None, 6]], columns=list("ABC"))

In [3]: df['B']._parent_axis
Out[3]: 0

In [4]: df.loc[1]._parent_axis
Out[4]: 0
```

Reply:
@sfc-gh-joshi should we override this code? BTW, in indexing, we never use `_parent_axis` before.

Reply:
In our code base, we never set `_parent_axis = 1`.

Reply:
I think we should leave it as-is. We don't use it, but upstream modin uses it in `DataFrame._getitem_column` (which we don't use since we use a different code path for indexing).

Reply:
we now set _parent_axis=1 correctly: https://github.com/snowflakedb/snowpark-python/pull/2210